frappe/desk/page/leaderboard/leaderboard.css

```diff
@@ -55,3 +55,7 @@
 	max-width: 75px;
 }
 
+.leaderboard-search {
```

Comment:
Add `form-group` class [here](https://github.com/prssanna/frappe/blob/leaderboard-fix/frappe/desk/page/leaderboard/leaderboard.js#L190) instead of adding separate style for leaderboard search.

It will help to maintain a consistent style after a refactor like [this](https://github.com/frappe/frappe/pull/9081).